### PR TITLE
NeurIPS2020 L2RPN Challenge

### PR DESCRIPTION
**Pull request by YuechengLiu**

### CODE REVIEW

File: examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/utils.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+selected_feature = [
+    'loads_q', 'loads_v', 'prods_q', 'prods_v', 'rho', 'line_status',
+    'hour_of_day', 'month'
+]
+inference_info = np.load('./saved_files/inference_info.npz')
+col = inference_info['col']
+mean = inference_info['mean']
+#print("MMMMMMMMMMMM", mean[-:])
```
Comment (TomorrowIsAnOtherDay) [inline examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/utils.py]: please remove the commented lines.

File: examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## NeurIPS2020 L2RPN Challenge
+
+The **PARL** team gets the first place for both tracks (Robustness Track and Adaptability Track) in *NeurIPS2020 Learning-to-Run-a-Power-Network* challenge!
```
Comment (TomorrowIsAnOtherDay) [inline examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/README.md]: all tracks ( both XXXtrack and XXX track)

File: examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## NeurIPS2020 L2RPN Challenge
+
+The **PARL** team gets the first place for both tracks (Robustness Track and Adaptability Track) in *NeurIPS2020 Learning-to-Run-a-Power-Network* challenge! 
+
+<p align="center">
+<img src="images/l2rpn.jpeg" alt="PARL" height="300" />
+</p>
+
+## Codes for evaluation
```
Comment (TomorrowIsAnOtherDay) [inline examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/README.md]: Note: We release the code with trained policy for our top submission at the moment.  Regarding the training code, we are still working on a technical report and will release it as soon as possible.

Comment (YuechengLiu) [reply examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/README.md]: updated

**Review by zenghsh3 — APPROVED** [review]
LGTM

**Pull request merged**